backend/src/main/java/org/backend/ridinglesson/Ridinglesson.java

```diff
@@ -0,0 +1,11 @@
+package org.backend.ridinglesson;
+
+public record Ridinglesson(
+        String id,
+        String ridinginstructor,
+        String ridingtype,
+        String horse,
+        String Date,
```

Comment:
hier sollten die Variabeln anfangs kleingeschrieben sein, damit es klar ist, dass es keine Klasse ist
```suggestion
        String date,
```